app/components/post_body/post_body.js

```diff
@@ -116,9 +116,9 @@ export default class PostBody extends PureComponent {
         const {height} = event.nativeEvent.layout;
         const {showLongPost} = this.props;
 
-        if (!showLongPost) {
+        if (!showLongPost && height >= this.state.maxHeight) {
             this.setState({
-                isLongPost: height >= this.state.maxHeight,
+                isLongPost: true,
```

Comment:
This won't reset `isLongPost` to `false` if the height of the post is reduced after the initial post size exceeded the max height.

Reply:
right, when editing a post.. I'll take a look and see cause this was causing some unnecessary  re-renders